runbotics/runbotics-orchestrator-ui/src/src-app/views/AIAssistantView/AIAssistantView.tsx

```diff
@@ -1,15 +1,28 @@
 import React, { VFC } from 'react';
 
 
+
+import useAuth from '#src-app/hooks/useAuth';
 import useTranslations from '#src-app/hooks/useTranslations';
 
+import { redirectToWebsiteRoot } from '#src-app/utils/navigation';
+
 import { StyledIFrame, StyledPage } from './AIAssistantView.styles';
 
+
 const AIAssistantView: VFC = () => {
     const { translate } = useTranslations();
+    const { user } = useAuth();
+
 
     const pageTitle = translate('AIAssistant.Title');
 
+    const isTenantDefault = user.tenant.id === 'b7f9092f-5973-c781-08db-4d6e48f78e98'; // temporary additional check
```

Comment:
This is fine - we will remove it in the future, if we decide to support other tenants

Reply:
one question: is this tenant ID common in all evironments (local, dev, qas, prod, com)?